tests/functional/test_loaded_model.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
+import grpc
+import pytest
+
+from tritonclient.grpc import service_pb2
+from tritonclient.grpc import service_pb2_grpc
+import tritonclient.grpc as grpcclient
+from model_config import MODEL_CONFIG
+PORT=8111
+
+
+@pytest.mark.parametrize("model", MODEL_CONFIG.keys())
```

Comment:
add tests for GPU with a marker like in https://github.com/openvinotoolkit/docker_ci/blob/master/tests/conftest.py#L543